components/index.js

```diff
@@ -0,0 +1,10 @@
+export { default as Button } from './button';
+export { default as Dashicon } from './dashicon';
+export { default as FormToggle } from './form-toggle';
+export { default as IconButton } from './icon-button';
+export { default as Panel } from './panel';
+export { default as Placeholder } from './placeholder';
+export { default as Spinner } from './spinner';
+export { default as Toolbar } from './toolbar';
+
+export { default as withFocusReturn } from './higher-order/with-focus-return';
```

Comment:
How do we make this work like `wp.components.something` behind the scenes?

I think this could be solved separately, in any case.